### DISCUSSION TITLE
Way to start this project in local for contribution

### DISCUSSION THREAD

**Discussion by pandeysuresh06** [apolloconfig/apollo#4318] [Q&A] [answered]
Hi Team,

I downloaded the code but maven is not able to resolve the dependencies. Is there any guide to make the application work in the local environment.

**Comment by Anilople** [comment, marked as answer]
use
```bash
mvn clean compile
```
check the dependencies first.

There is a https://www.apolloconfig.com/#/zh/development/apollo-development-guide may can help you?
